com.inform7.IDE.yml

```diff
@@ -123,16 +123,16 @@ modules:
       - '*.la'
     sources:
       - type: archive
-        url: "http://sourceforge.net/projects/modplug-xmms/files/libmodplug/\
-          0.8.9.0/libmodplug-0.8.9.0.tar.gz"
-        sha1: 9d18572f85fdd95091befa972ee5d94139dcefde
+        url: https://downloads.sourceforge.net/project/modplug-xmms/libmodplug/0.8.9.0/libmodplug-0.8.9.0.tar.gz
+        sha256: 457ca5a6c179656d66c01505c0d95fafaead4329b9dbaa0f997d00a3508ad9de
```

Comment:
Just a note to myself: I verified that the file downloaded from the _new_ URL matches the _old_ checksum.